components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/context/model/Flow.java

```diff
@@ -38,6 +38,8 @@ public class Flow {
                 EnumSet.of(InitiatingPersona.ADMIN, InitiatingPersona.APPLICATION));
         FLOW_DEFINITIONS.put(Name.PROFILE_UPDATE,
                 EnumSet.of(InitiatingPersona.ADMIN, InitiatingPersona.APPLICATION, InitiatingPersona.USER));
+        FLOW_DEFINITIONS.put(Name.SCIM_BULK_RESOURCE_UPDATE,
```

Comment:
If this flow is group update this should be group update

Reply:
Fixed 8aeb98a35dd58bc2840c6e9ee95ecd30a684245b